tests/test_validation_exception_handling.py

```
@@ -0,0 +1,83 @@
+
+from ddb_single.table import Table
+
+
+class TestValidationExceptionHandling(unittest.TestCase):
+    def test_query_validation_exception_is_reraised(self):
+        table = Table(
+            table_name="test_table",
+            endpoint_url="http://localhost:8000",
+            region_name="us-west-2",
+            aws_access_key_id="fakeMyKeyId",
+            aws_secret_access_key="fakeSecretAccessKey",
```

Ruff (0.15.12)

[error] Possible hardcoded password assigned to argument: "aws_secret_access_key"

(S106)